### PR TITLE
Images and get markdown from gdocs

### PR DESCRIPTION
**Pull request by jdemonasterio**
- Init PR to check renderization of Latex formulas, images and general Markdown synthax.results Latex .
- Preview post.
- Final English grammar and syntax check.

### CODE REVIEW

File: _src/_posts/2017-09-11-importance-of-serving-ads.markdown [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+[Jampp] [jampp] helps customers boost their mobile sales with performance marketing. One of the most challenging problems of the industry is the impact quantification an ad has on users. A poorly tackled problem is understanding when a user has been affected enough by ads, and why acquiring clicks is not always best aligned with the advertiser's interests.
+We know that, sometimes, serving more impressions to users will result in CPA increase, since we can not influence their behavior any more. The key question is: how many ads are enough? 
+
+<!--excerpt.end→
+
+##Introduction
+
+Performance advertising is relevant to advertisers that want to have a quantifiable measure of their ROI. This means that the investment in advertising has to reach measurable goals such as increased number of in-app activity or app downloads. Then, the payment is directly measured by the value brought by the _events_ caused by ads. 
+
+RTB ad platforms all enforce value prediction at the user and creative level. These typically take into consideration history of information from the auction itself or from the advertiser's campaigns, but little effort is put on the user's history of impressions.
```
Comment (santijampp) [inline _src/_posts/2017-09-11-importance-of-serving-ads.markdown]: history of information from the auction itself  -> auction's information history
user's history of impressions -> user's impression history
?

File: _src/_posts/2017-09-11-importance-of-serving-ads.markdown [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+<!--excerpt.end→
+
+##Introduction
+
+Performance advertising is relevant to advertisers that want to have a quantifiable measure of their ROI. This means that the investment in advertising has to reach measurable goals such as increased number of in-app activity or app downloads. Then, the payment is directly measured by the value brought by the _events_ caused by ads. 
+
+RTB ad platforms all enforce value prediction at the user and creative level. These typically take into consideration history of information from the auction itself or from the advertiser's campaigns, but little effort is put on the user's history of impressions. 
+
+Today, advertisers spend millions in marketing budget among different partners. They have to determine who takes the credit for helping the user convert, in other words, which partner is the action _attributed_ to.
```
Comment (santijampp) [inline _src/_posts/2017-09-11-importance-of-serving-ads.markdown]: helping the user convert -> making the user convert?

File: _src/_posts/2017-09-11-importance-of-serving-ads.markdown [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+author: jdemonasterio
+---
+
+
+<!--excerpt.start-->
+
+[Jampp] [jampp] helps customers boost their mobile sales with performance marketing. One of the most challenging problems of the industry is the impact quantification an ad has on users. A poorly tackled problem is understanding when a user has been affected enough by ads, and why acquiring clicks is not always best aligned with the advertiser's interests.
+We know that, sometimes, serving more impressions to users will result in CPA increase, since we can not influence their behavior any more. The key question is: how many ads are enough? 
+
+<!--excerpt.end→
```
Comment (jdemonasterio) [inline _src/_posts/2017-09-11-importance-of-serving-ads.markdown]: fake comment, trying to re-open this PR as per stackoverflow help

https://stackoverflow.com/questions/12674304/github-reopening-a-merged-pull-request

Comment (jdemonasterio) [inline _src/_posts/2017-09-11-importance-of-serving-ads.markdown]: foo

**Comment by mbonamico** [comment]
.

File: _src/_posts/2017-09-11-importance-of-serving-ads.markdown (hunk 1 above)
Comment (jdemonasterio) [reply _src/_posts/2017-09-11-importance-of-serving-ads.markdown]: Semi correction. Notice here that `information history` is a modifier for both the `auction` object and the `advertiser's campaigns` object. Maybe there's a better, correct and simple way to convey the idea. 
I'm modifying this to : `information history from the auction itself, or from the advertiser's campaigns`

Comment (jdemonasterio) [reply _src/_posts/2017-09-11-importance-of-serving-ads.markdown]: Think that we use both the bid's context (user id, ip, user agent, publisher, etc) and creative info (advertiser, creative_id, action,etc) to price the bid.

File: _src/_posts/2017-09-11-importance-of-serving-ads.markdown (hunk 2 above)
Comment (jdemonasterio) [reply _src/_posts/2017-09-11-importance-of-serving-ads.markdown]: Well, that's kind of the crux of this post. Do ads make the user convert or help him convert? How many ads do we need? Do we really have causality? I'm accepting the suggestion but I think we can use another (better) word.

**Pull request merged**